tools/LLVMOptimizer.h

```diff
@@ -0,0 +1,5 @@
+#include "llvm/Passes/PassBuilder.h"
```

Comment:
You don't need these includes. Plus add `#pragma once` & license header.

Reply:
If I delete "PassBuilder.h" I can't build it locally. 
I suspect it's my CMakeList.txt problem. Let me check it.

Reply:
At the error message, it says 
"/home/runner/work/alive2/alive2/tools/LLVMOptimizer.h:1:10: fatal error: 'llvm/Passes/PassBuilder.h' file not found
#include "llvm/Passes/PassBuilder.h""

I don't know it can't find the header. Is it any problem that I should specify something in the Cmake.txt?

Reply:
You just need to do a forward declaration. These are only pointers.

Reply:
Hi Nuno, I am sorry. I don't think I get you... In my newest push, I removed function and added #pragma once and license. >_<

Reply:
Oh Wait, better idea: This file is LLVM related so I should place it in /llvm_util/ rather /tool/
What do you think?